<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtlMovieMainWindow</class>
 <widget class="QMainWindow" name="QtlMovieMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtlMovie</string>
  </property>
  <property name="windowIcon">
   <iconset resource="QtlMovie.qrc">
    <normaloff>:/images/qtlmovie-logo.png</normaloff>:/images/qtlmovie-logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="layoutCentral">
    <item row="0" column="0" colspan="2">
     <widget class="QTableWidget" name="taskList">
      <property name="columnCount">
       <number>2</number>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column/>
      <column/>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="labelRemainingTime">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QProgressBar" name="progressBarEncode">
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QtlPlainTextLogger" name="log">
      <property name="textInteractionFlags">
       <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="buttonStart">
      <property name="text">
       <string>Start ...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>679</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionTranscode"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionReleaseNotes"/>
    <addaction name="actionSearchNewVersion"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutMediaTools"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionInputFileProperties"/>
    <addaction name="actionTestAudio"/>
    <addaction name="actionDvdDecrypter"/>
    <addaction name="separator"/>
    <addaction name="actionClearLog"/>
    <addaction name="actionLogDebug"/>
    <addaction name="actionSaveLog"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About QtlMovie</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionTranscode">
   <property name="text">
    <string>Transcode ...</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings ...</string>
   </property>
  </action>
  <action name="actionInputFileProperties">
   <property name="text">
    <string>Input File Properties ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionClearLog">
   <property name="text">
    <string>Clear Log</string>
   </property>
  </action>
  <action name="actionSaveLog">
   <property name="text">
    <string>Save Log ...</string>
   </property>
  </action>
  <action name="actionTestAudio">
   <property name="text">
    <string>Test Audio ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionAboutMediaTools">
   <property name="text">
    <string>About Media Tools</string>
   </property>
  </action>
  <action name="actionDvdDecrypter">
   <property name="text">
    <string>DVD Decrypter ...</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help ...</string>
   </property>
  </action>
  <action name="actionLogDebug">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Log Debug</string>
   </property>
  </action>
  <action name="actionSearchNewVersion">
   <property name="text">
    <string>Search New Version</string>
   </property>
  </action>
  <action name="actionReleaseNotes">
   <property name="text">
    <string>Release Notes ...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QtlPlainTextLogger</class>
   <extends>QPlainTextEdit</extends>
   <header>QtlPlainTextLogger.h</header>
   <slots>
    <slot>saveToFile()</slot>
    <slot>setDebugMode(bool)</slot>
    <slot>save()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="QtlMovie.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>QtlMovieMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>QtlMovieMainWindow</receiver>
   <slot>about()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSettings</sender>
   <signal>triggered()</signal>
   <receiver>QtlMovieMainWindow</receiver>
   <slot>editSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClearLog</sender>
   <signal>triggered()</signal>
   <receiver>log</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>478</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSaveLog</sender>
   <signal>triggered()</signal>
   <receiver>log</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>466</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAboutMediaTools</sender>
   <signal>triggered()</signal>
   <receiver>QtlMovieMainWindow</receiver>
   <slot>aboutMediaTools()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDvdDecrypter</sender>
   <signal>triggered()</signal>
   <receiver>QtlMovieMainWindow</receiver>
   <slot>startDvdDecrypter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionHelp</sender>
   <signal>triggered()</signal>
   <receiver>QtlMovieMainWindow</receiver>
   <slot>showHelp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLogDebug</sender>
   <signal>toggled(bool)</signal>
   <receiver>log</receiver>
   <slot>setDebugMode(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>466</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSearchNewVersion</sender>
   <signal>triggered()</signal>
   <receiver>QtlMovieMainWindow</receiver>
   <slot>searchNewVersion()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionReleaseNotes</sender>
   <signal>triggered()</signal>
   <receiver>QtlMovieMainWindow</receiver>
   <slot>showReleaseNotes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>startDvdDecrypter()</slot>
  <slot>editSettings()</slot>
  <slot>aboutMediaTools()</slot>
  <slot>about()</slot>
  <slot>showHelp()</slot>
  <slot>transcodingStarted()</slot>
  <slot>transcodingStopped(bool)</slot>
  <slot>transcodingProgress(QString,int,int,int,int)</slot>
  <slot>searchNewVersion()</slot>
  <slot>showReleaseNotes()</slot>
 </slots>
</ui>
